app/controllers/hyrax/dashboard/collections_controller.rb

```diff
@@ -192,10 +197,6 @@ def files
         render json: result
       end
 
-      def search_builder_class
```

Comment:
I'm a little curious about this going away.  But will need to do some more investigating on my local machine.

Reply:
It's been a few weeks but... this class overrides the method to change the search builder but configuring it in the blacklight config (as above) is a happier path for blacklight. Downstream code shouldn't see any difference because blacklight's going to provide `search_build_class` with the right value 🤷‍♂️